packages/frontend/src/hooks/dashboard/useDashboard.tsx

```diff
@@ -304,6 +304,9 @@ export const useCreateMutation = (
                           }
                         : undefined,
                 });
+                history.push(
+                    `/projects/${projectUuid}/dashboards/${result.uuid}/edit`,
+                );
```

Comment:
this might be a problem in places where you want to use useMutation but you don't want to have a redirect.

also, history used here is a browser internal and not a history from the react-router.

ideally, you should do the redirect after the mutation succeeds in the component where you are using the hook